apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/TableSchemaBuilder.cs

```diff
@@ -40,10 +41,30 @@ public sealed class TableSchemaBuilder : IEnumerable
         public void Add(TableFieldSchema field)
         {
             GaxPreconditions.CheckNotNull(field, nameof(field));
-
+            _tableFields[field.Name] = field;
             _fields.Add(field);
         }
 
+        /// <summary>
+        /// Modifies the given fields as per the specified action.
+        /// </summary>
+        /// <param name="fieldName">The name of the field to be modified. Must not be null.</param>
+        /// <param name="modifyField">An Action to be performed on the given field.</param>
+        /// <returns>A <see cref="TableSchemaBuilder"/> with the modified field.</returns>
+        public TableSchemaBuilder ModifyField(string fieldName, Action<TableFieldSchema> modifyField)
```

Comment:
I'd suggest either just `action` or `fieldModifier` as names... something more like a noun, basically. @amanda-tarafa may have other suggestions.

Reply:
updated it to fieldModifier @amanda-tarafa let me know if you have any other suggestions.